ResearchKit/Localized/en.lproj/ResearchKit.strings

```diff
@@ -180,6 +180,9 @@
 "MEASURING_UNIT_CM" = "cm";
 "MEASURING_UNIT_FT" = "ft";
 "MEASURING_UNIT_IN" = "in";
+"MEASURING_UNIT_KG" = "kg";
+"MEASURING_UNIT_LBS" = "lbs";
```

Comment:
If changing the default to "lb" then let's provide a way for a developer to override this for cases they need to use "lbs".  This is particularly important for patient-facing apps where the colloquial usage is what's expected by most users.

Reply:
I kindly disagree.

I wrote *colloquial*, but I meant to say *incorrect*. It's like if Apple were to use slang, or use incorrect English usages such as *your* in place of *you're* on their official iOS translations just because many people do that mistake. I think we should promote correct English within serious apps such ResearchKit-based ones. I think mostly no people will be put off by an app displaying `lb` instead of `lbs`, even if they're more used to the latter. 

See these answers for further clarification: https://english.stackexchange.com/questions/113142/correct-usage-of-lbs-as-in-pounds-of-weight